extensions/wikidata/module/langs/translation-en.json

```diff
@@ -80,6 +80,8 @@
     "wikibase-preview/new-id": "new item",
     "wikibase-account/dialog-header": "$1 account",
     "wikibase-account/explain-log-in": "Logging in to <a href=\"$1\" target=\"_blank\">$2</a> lets you to upload edits directly from OpenRefine.",
+    "wikibase-account/explain-bot-passwords": "You are encouraged to login with <a href=\"$1\" target=\"_blank\">bot passwords</a>, see this <a href=\"https://github.com/OpenRefine/OpenRefine/wiki/MediaWiki-Bot-Passwords\" target=\"_blank\">wiki</a> to learn how to get one.",
```

Comment:
Shouldn't this point instead to Mediawiki's help pages explaining bot passwords? Instead of our wiki?  It would be better if we didn't have to maintain this page, I think.

Reply:
We need a wiki to explain which "Applicable grants" are needed, merely the link to the MediaWiki's manual seems not enough.

Reply:
Wikidata is a wiki.  Wikipedia is a wiki.  Both have Special_Pages, manual pages and Help pages.  I think we should just point to use one of them.  I don't think this is entirely OpenRefine specific.  Here's a number of them, there are probably more on mediawiki.org that I am not aware of.

https://en.wikipedia.org/wiki/MediaWiki:Botpasswords-summary
https://www.mediawiki.org/wiki/Manual:Pywikibot/BotPasswords
https://www.wikidata.org/wiki/Wikidata:Creating_a_bot/en
https://www.wikidata.org/wiki/Special:BotPasswords
https://www.wikidata.org/wiki/Special:SpecialPages

https://www.wikidata.org/wiki/Wikidata:Tools/OpenRefine

I think one of those might be used instead and then expanded upon within it?
I don't know.
@wetneb probably would or have contacts that do know.

Reply:
I think it makes sense to have OpenRefine-specific documentation, at least for the Applicable grants.